proteobench/modules/dda_quant/parse_settings.py

```diff
@@ -39,3 +44,7 @@ def __init__(self, input_format:str):
         self.min_count_multispec = parse_settings["general"]["min_count_multispec"]
         self.species_expected_ratio = parse_settings["species_expected_ratio"]
     
+
+def parse_settings(input_format:str) -> Settings:
```

Comment:
Instead of having a class to parse the setting, we could have a `Settings` dataclass instance returned by simple function. Then the type is defined. See `interfaces.py`